com.unity.netcode.gameobjects/Components/NetworkTransform.cs

```diff
@@ -875,10 +873,7 @@ protected virtual void Update()
 
             if (CanCommitToTransform)
             {
-                if (m_CachedIsServer)
```

Comment:
This was there for historical reason. Now that we can write to netvar client side, this shouldn't be here anymore